src/molecules/modal/Modal.js

```diff
@@ -1,41 +1,68 @@
 // @vue/component
 export default {
   props: {
+    /**
+     * Modal name
+     */
     name: {
       type: String,
       required: true
     },
+    /**
+     * Modal heading text
+     */
     heading: {
       type: String,
       default: null
     },
+    /**
+     * Indicate if clicking outside modal can emit `close` action
+     */
     closeOnBackgroundClick: {
       type: Boolean,
       default: true
     },
+    /**
+     * Indicate if Esc key can emit `close` action
+     */
     closeOnEsc: {
       type: Boolean,
       default: true
     },
-    maxWidth: {
-      type: Number,
-      default: null
+    /**
+     * Modal main transition effect name
+     */
+    modalMainTransition: {
+      type: String,
+      default: 'fade'
```

Comment:
Can we add this transition type and `modalContentTransition` to knobs and use dynamically? 
The knob can be a list of `fade`, `slide-down`, `slide-right`

Reply:
Good idea, added.